tools/util_generate_hex_from_json.cpp

```diff
@@ -0,0 +1,148 @@
+//
+// Created by Eric Passmore on 1/14/23.
+// Purpose: command line option to generate hex code from JSON
+//   may be used to generate serialization tests cases in other languages and external packages
+//
+
+#include "../src/abieos.h"
```

Comment:
Use `#include <abieos.h>`

Reply:
You are using the C interface of abieos instead of the C++ interface of abieos. For these two little utils I guess it really doesn't matter that much. I don't see it is worth asking you to rewrite it to use the C++ interface.